beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -221,23 +212,78 @@ func (s *Server) GetBlockAttestations(w http.ResponseWriter, r *http.Request) {
 			return
 		}
 	}
+	resp := &structs.GetBlockAttestationsResponse{
+		Data:                atts,
+		ExecutionOptimistic: isOptimistic,
+		Finalized:           s.FinalizationFetcher.IsFinalized(ctx, root),
+	}
+	httputil.WriteJson(w, resp)
+}
+
+// GetBlockAttestationsV2 retrieves attestation included in requested block.
+func (s *Server) GetBlockAttestationsV2(w http.ResponseWriter, r *http.Request) {
+	ctx, span := trace.StartSpan(r.Context(), "beacon.GetBlockAttestationsV2")
+	defer span.End()
+
+	blk, isOptimistic, root := s.blockData(ctx, w, r)
+	consensusAtts := blk.Block().Body().Attestations()
+
+	var atts = make([]interface{}, len(consensusAtts))
+	v := version.String(blk.Block().Version())
+
+	if v == version.String(version.Electra) {
```

Comment:
i think it'll be better to do blk.Block().Version >= version.Electra or something like that